### DISCUSSION TITLE
Flows disappearing?

### DISCUSSION THREAD

**Discussion by kmythen** [postmanlabs/postman-flows#139] [Defect]
Hi everyone, been using the flows for sometime now and really enjoying them. However, I have ran into a problem where my flows keep deleting/reverting to previous versions. Is this a bug?

For example, I could build a complex flow, then click onto another section of Postman, and when I return to the flow, all of my work is gone ;( 

Thank you!

**Comment by saswatds** [comment]
Hi @kmythen, this is really unfortunate. It should not be happening. We will immediately investigate in our logs the root cause of this problem. 
There is one issue we are aware of when saving is failing is when `Assign Variable` or `Create Variable` has a variable name that contains the characters `$` or `.` in its name. Would this be a case for you?

**Reply by kmythen** [reply]
Hi @saswatds, thanks for getting back to me! 

Aww yes, I do have one assigned variable that has a `.` in it, so this would explain it. I would have thought it would just fail to save that variable, but it seems to be anything after that variable has been set.

It's good to know you guys are aware though and will keep an eye out for a fix.

Thanks again!

**Reply by saswatds** [reply]
@kmythen We deployed a fix for this yesterday. Could you help verify that it fixed your issue?

**Reply by kmythen** [reply]
Hi @saswatds, tested and working, all resolved - thank you for resolving so quickly :)

**Comment by kmythen** [comment]
Hi @saswatds, hope your doing well, just a quick update - seems like the issue has returned. 

My flow changes are not saving, but also have noticed some examples of `.` changing in variables, seems like the same issue.

Cheers!

**Reply by saswatds** [reply]
@kmythen It should work now. We had to revert the fix for a day because we encountered another issue that got introduced with this fix. We have now redeployed the fix.

**Reply by kmythen** [reply]
Thanks @saswatds :)
